benches/bench.rs

```diff
@@ -170,3 +170,51 @@ fn size_22_4m(b: &mut test::Bencher) {
 fn size_23_8m(b: &mut test::Bencher) {
     bench_size(b, 8 * 1024 * 1024);
 }
+
+macro_rules! gen_poll_test {
```

Comment:
This is what I originally did for the other benchmark tests -- but @pcwalton didn't like the use of macros here...